src/sgp/VObject.cc

```diff
@@ -369,12 +364,12 @@ static void DumpVObjectInfoIntoFile(const char* filename, BOOLEAN fAppend)
 	UINT32 uiUniqueID = 0;
 	for (SGPVObject const* i = gpVObjectHead; i; i = i->next_)
 	{
-		char const* const Name = i->name_;
-		char const* const Code = i->code_;
+		char ST::string& Name = i->name_;
+		char ST::string& Code = i->code_;
```

Comment:
I doubt this compiles, but `SGP_VIDEO_DEBUGGING` was already broken. As far as I am concerned it should be next on the chopping block.

Reply:
Always happy to delete code. 👍🏾